src/sagemaker/fw_utils.py

```diff
@@ -198,6 +198,19 @@ def tar_and_upload_dir(session, bucket, s3_key_prefix, script,
     return UploadedCode(s3_prefix='s3://%s/%s' % (bucket, key), script_name=script_name)
 
 
+def upload_file(session, bucket, s3_key_prefix, file, kms_key=None):
```

Comment:
I am not sure this function will be necessary anymore.

Reply:
correct